api/user-recommendations/config/routes.json

```diff
@@ -0,0 +1,13 @@
+{
+  "routes": [
+    {
+      "method": "GET",
+      "path": "/users/user-recommendations",
+      "handler": "user-recommendations.recommend",
+      "config": {
+        "auth": false,
```

Comment:
Does this mean anyone can call this endpoint? I think each user should only be able to get his/her recommendations.